models/xero__general_ledger.sql

```diff
@@ -1,3 +1,41 @@
+/*
+in order to generate trancking_category_name as a column we set a dictionary
+dictionary has the following appareance{'tracking_category_id' : [tuple of available id], 'tracking_category_name': [tuple of available names]}
```

Comment:
I'd maybe switch `appearence` -> `structure`?

Reply:
Also indent the objcet if possible?

Reply:
Sure!